packages/react/src/components/Popover/index.tsx

```diff
@@ -22,31 +22,39 @@ export const Popover = ({
 }: PopoverProps) => {
   const popoverRef = useRef<HTMLDivElement>(null);
   useHandleClickOutside(parentRef, onClickOutside);
-  const [topPosition, setTopPosition] = useState(0);
+  const [topPosition, setTopPosition] = useState<number | undefined>();
 
   useLayoutEffect(() => {
     const handleWindowResize = () => {
```

Comment:
Just one last improvement. Can you throttle this callback in order to exec it only each, lets say, 500ms ?

Reply:
[fixup! ✨(react) position top datepicker](https://github.com/openfun/cunningham/pull/133/commits/4d65cfd8aa0bb460ad977b2067e394bca8b32b4d)